### ISSUE TITLE
Migrations are not up to date

### ISSUE THREAD

**Issue by Gwildor** [stephenmcd/django-forms-builder#172]
I've installed version 0.12.2, and ran `makemigrations` after I had created a custom content type for the FeinCMS page model. Django saw changes and created a migration (see contents below), and worse, added the new migrations for forms builder as a dependency of my own app's migration. Because of this, I got a "migration does not exist" error running the migrations on production.

The following migration is created:

```python
operations = [
    migrations.AlterField(
        model_name='form',
        name='email_from',
        field=models.EmailField(verbose_name='From address', help_text='The address the email will be sent from', max_length=254, blank=True),
    ),
    migrations.AlterField(
        model_name='form',
        name='sites',
        field=models.ManyToManyField(default=[1], editable=False, to='sites.Site', related_name='forms_form_forms'),
    ),
]
```

**Comment by stephenmcd** [comment]
Duplicate #167

**Comment by Gwildor** [comment]
Well, it's partially a duplicate. Using the version from master fixed the issue with the `sites` field, but the issue with the `email_from` field's `max_length` change is still there. But that is an issue for Django 1.8, and should be in a "Django 1.8 support" ticket I guess.

**Comment by Gwildor** [comment]
Please note that this is still an issue, which I unfortunately fell for again on anther project. Django made a new migration for forms, and that again was referenced in one of my project apps. This of course failed when deploying.

**Comment by stephenmcd** [comment]
You're right, the email field automatically changed length in Django 1.8 so this wasn't resolved.

It's fixed in a18862031b61cc8ab4cae74b2bace47ec6c6bd3b

Thanks
